### ISSUE TITLE
$ANDROID_HOME env variable collides with expected default from Flutter

### ISSUE THREAD

**Issue by thunderbiscuit** [bitcoindevkit/bdk-old#42]
Setting up Flutter on my Ubuntu machine I'm getting a small problem with what I think is the overwrite of an environment variable with `bdk`.

Flutter comes with this tool called flutter doctor which checks your environment and displays a report of the status of your Flutter installation. In my case, it's telling me it can't find the Android sdk, and that is because it is using `$ANDROID_HOME`, where it expects to find the sdk, and I have redefined that `$ANDROID_HOME` environment variable as per the readme in bdk. The sdk is actually one level down at `/home/username/Android/Sdk/`, and so when I change my variable in my `.bashrc`, it fixes my issue and the flutter doctor report comes back clean. 

I am posting this here as an issue because I'm not sure of how that environment variable comes into play for bdk, and whether the readme should/could be adapted to not mess with those, and whether that environment variable could create other problems elsewhere for other folks.

**Comment by notmandatory** [comment]
I ran into this on Linux too and I think the README has the right instructions: 
```
Android (Linux):

export ANDROID_HOME=$HOME/Android
export ANDROID_NDK_HOME=$ANDROID_HOME/Sdk/ndk/<ndk version, eg. 21.0.6113669>
```
That's the same on your env? To answer your other question, the android SDK is used by `cargo ndk` to build the android libs, see the build-libs.sh file.  And more details about cargo ndk here: https://github.com/bbqsrc/cargo-ndk

**Comment by thunderbiscuit** [comment]
Yes basically the fact that I had followed the readme and set my `$ANDROID_HOME=$HOME/Android` in my `.bashrdc` prevented flutter from finding the sdk.

I had to switch things slightly to:
```sh
export ANDROID_HOME=$HOME/Android/Sdk
export ANDROID_NDK_HOME=$ANDROID_HOME/ndk/<ndk version, eg. 21.0.6113669>
```
for Flutter to work. But I assume that will impact `bdk` no? Am I messing up one by fixing the other?

**Comment by notmandatory** [comment]
OK now I understand. I agree with your fix above.

**Comment by thunderbiscuit** [comment]
Now fixed with #43.
